src/swarmus_ros_simulation/launch/simple_swarm.launch

```diff
@@ -2,11 +2,14 @@
 <!-- This launch files spawns a swarm of 5 fully functional robots -->
 
 <launch>
+    <arg name="interloc_refresh_rate" default="5.0"/> 
```

Comment:
Is it normal that the default value is 10 Hz in the simulated_agent but 5 Hz here?